Shared/ThresholdKeyView.swift

```diff
@@ -531,25 +569,31 @@ struct ThresholdKeyView: View {
                         Spacer()
                         Button(action: {
                             Task {
-                                let shareStore = try! await threshold_key.generate_new_share()
-                                let index = shareStore.hex
+                                do {
+                                    let shareStore = try await threshold_key.generate_new_share()
```

Comment:
Are we intentionally discarding the newly generated share, or should we be saving it?

Reply:
Hmm, this one can left for another PR.